src/constructorio.js

```diff
@@ -169,6 +192,11 @@ class ConstructorIO {
         this.options.testCells = testCells;
       }
 
+      if (sessionId) {
+        this.options.sessionId = sessionId;
```

Comment:
Not a huge fan of saving the `sessionId` to the options. We'll have two places the value will reside, but we should ideally be using the value from localStorage/cookies.

I'm guessing this was mostly used for testing, but if we add `set_session_id` (mentioned above) and `get_session_id` to the identity module, we shouldn't need to keep track of this in multiple places.

Reply:
But this is already in options, I'm just updating it with the new value 🤔
Am I missing something?

Reply:
Ah, I completely missed that somehow. Please ignore me! 😅